RocketChat/Rocket.Chat.ReactNative
# [FIX] Reactions list isn't scrollable

## Proposed changes


In a reaction list when there are many reactions in `All` tab or showing the users and the action sheet isn't completely opened, there is a visual bug that depending on the platform, the behavior of the scroll is different and can happen that the scroll isn't scrollable.

## Issue(s)	


## How to test or reproduce


- Go to a channel there are multiple reactions
- do a long press on the emoji reaction
- let the action sheet on the first step of openning (don't pull the action sheet to bigger)
- try to scroll, you should see all the persons reactions before

## Screenshots

### Android
#### Before
https://user-images.githubusercontent.com/47038980/216995617-28ec5d06-13f2-4581-80d6-851a501f7e76.mov

#### After
https://user-images.githubusercontent.com/47038980/216995686-957a183c-41f5-4f01-b6d3-d0eb52e2a54f.mov

### iOS
#### Before
https://user-images.githubusercontent.com/47038980/216995731-4b9c08aa-cd09-4647-a847-f0b07ff0a1b0.mov

#### After
https://user-images.githubusercontent.com/47038980/216996240-580e1093-bf9a-42ce-9f51-a2229ef676a3.mov



## Types of changes




## Checklist



## Further comments

[NATIVE-206]

[NATIVE-206]: https://rocketchat.atlassian.net/browse/NATIVE-206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
